Makefile.PL

```diff
@@ -49,7 +49,7 @@ my %requires = (
   'DBI'                                 => '1.614',
   'DBIx::Connector'                     => 0,
   'DBIx::Class'                         => 0,
-  'DBIx::Class::Helpers'                => 0,
+  'DBIx::Class::Helpers'                => '== 2.034002',
```

Comment:
this seems unrelated; why does this need pinning now, and if it's required why aren't the other dbix classes pinned?

Reply:
It is related in that any version above 2.034002 causes an error in the Model code Dylan wrote for sending data to STMO. The newer version came was released after the last time we generated a bmo-slim image. The Helpers module seems to be independent to the DBIx::Class module so the Class module does not need to be pinned currently.